driver/main.c

```diff
@@ -86,6 +86,8 @@ MODULE_AUTHOR("the Falco authors");
 #define _PAGE_ENC 0
 #endif
 
+#define TP_VAL_INTERNAL TP_VAL_MAX
```

Comment:
Canary value for internal events (whose `record_event_consumer` must skip tracepoint enabled check).